addons/arsenal/stringtable.xml

```diff
@@ -433,5 +433,17 @@
             <Italian>Impossibile aprire l'arsenale ACE</Italian>
             <Japanese>ACE 武器庫を開けません</Japanese>
         </Key>
+          <Key ID="STR_ACE_Arsenal_Mission">
+            <English>ACE Virtual Arsenal</English>
```

Comment:
Should be just "ACE Arsenal".

Reply:
:+1: The descriptions are placeholder / copy paste atm.